### ISSUE TITLE
Pretraining Hyperparameters

### ISSUE THREAD

**Issue by RobinLi7** [MAGICS-LAB/DNABERT_2#145]
May I ask what hyperparameters you used for pretraining? I’m curious about the learning rate, global and local batch sizes, optimizer, and scheduler settings you used.

**Comment by Zhihan1996** [comment]
Hey Robin, you can find the informations below.

<img width="971" alt="Image" src="https://github.com/user-attachments/assets/43058fba-067d-4211-863f-5a4735ceab2b" />
